torch/_numpy/random.py

```diff
@@ -64,7 +82,7 @@ def random_sample(size=None):
         size = ()
     dtype = _dtypes_impl.default_dtypes().float_dtype
     values = torch.empty(size, dtype=dtype).uniform_()
-    return array_or_scalar(values, return_scalar=size is None)
+    return array_or_scalar(values, return_scalar=size == ())
 
 
 @deco_stream
```

Comment:
Previous issue, but this decorator is not necessary, is it?

Reply:
On eager it is, why?

Reply:
nb. This comment refers to `rand`.